python/mlcroissant/mlcroissant/_src/operation_graph/operations/read.py

```diff
@@ -84,9 +85,14 @@ def _read_file_content(self, encoding_format: str, file: Path) -> pd.DataFrame:
         """Extracts the `source` file to `target`."""
         filepath = file.filepath
         if is_git_lfs_file(filepath):
-            download_git_lfs_file(file)
+            download_git_lfs_file(file, node=self.node)
         reading_method = _reading_method(self.node, self.fields)
-        with filepath.open("rb") as file:
+        file = filepath.open("rb")
```

Comment:
Do you have to manually close the file after it's used if you don't use the context manager anymore?

Reply:
Not sure -- if I keep the open in the while statement I shouldn't worry about closing it, right?